e2e/charts/newrelic-infrastructure/templates/daemonset-kubelet.yaml

```diff
@@ -98,6 +100,10 @@ spec:
                 fieldRef:
                   apiVersion: "v1"
                   fieldPath: "spec.nodeName"
+            - name: "NRIA_FEATURES"
+              value: "docker_enabled:false"
```

Comment:
Aw yess, I love this!

Maybe we could add a small comment so we don't forget why we do this :)

Reply:
couldn't this be a constraint if the future if we want to tweak that FF from the backend? I think local FF takes precedence over remote ones

Reply:
I think it could be hard to do this kind of detection from the backend, meaning: disabling docker `if kubernetes && (unprivileged || !processMetricsEnabled)`. Seems like a complex logic for the backend to handle.

Reply:
sure, what i meant is that we are going to override a parameter that the backend could (if needed for some reason) modify to manage ingestion. But actually we are disabling the feature so it wouldn't make sense that the backend enable this again. Just discard my comment :)

Reply:
I am not a super fan of this since I have the feeling that we should do the very same of nri-ecs:
```
integrations:
  - name: nri-ecs
    when:
      env_exists:
        ENABLE_NRI_ECS: "true"
    interval: 15s
```
However, if caos tell us it is safe I am fine with that 🙃